src/entities/task/model/slice.ts

```diff
@@ -0,0 +1,33 @@
+import { createEntityAdapter, createSlice } from "@reduxjs/toolkit";
+
+const sliceName = "taskList";
+
+export type Task = {
+    id: string;
+    title: string;
+    completed: boolean;
+};
+
+const tasksAdapter = createEntityAdapter<Task>({
+    selectId: (task: Task) => task.id,
+    sortComparer: (a: Task, b: Task) => a.title.localeCompare(b.title),
+});
+
+export const taskListSlice = createSlice({
+    name: sliceName,
+    initialState: tasksAdapter.getInitialState(),
+    reducers: {
+        addTask: (state, action) => {
```

Comment:
How about specify the type for `action.payload` explicitly? Also I suggest destructing the payload here